scripts/hypothesistooling.py

```diff
@@ -149,12 +150,19 @@ def create_tag_and_push():
         'git push ssh-origin --tags',
         'git push ssh-origin HEAD:master',
     ]:
-        subprocess.check_call([
-            'ssh-agent', 'sh', '-c',
-            'chmod 0600 deploy_key && ' +
-            'ssh-add deploy_key && ' +
-            command
-        ])
+        for retries_left in range(2, -1, -1):
+            try:
+                subprocess.check_call([
+                    'ssh-agent', 'sh', '-c',
+                    'chmod 0600 deploy_key && ' +
+                    'ssh-add deploy_key && ' +
+                    command
+                ])
+                break
```

Comment:
I’d probably have put this in an `else:` clause, so the body of the `try:` is as short as possible, but I don’t think that really matters here.